app/assets/javascripts/components/total_annotations.jsx

```diff
@@ -0,0 +1,53 @@
+var TotalAnnotations = React.createClass({
```

Comment:
So we have babel built in with `react-rails`. I probably should have realized this before in your previous JSX contributions, but for what it's worth, this entire class can be rewritten like so: https://gist.github.com/ptrikutam/1c5de886d5d38b541930807bef7a8279

If you want we can walk through that in detail, or you're welcome to refactor on your own.

Reply:
I really like your refactor. ES6 ❤️